build.psm1

```diff
@@ -1634,7 +1634,7 @@ esac
     # OpenSUSE 42.1 (13.2 might build but is EOL)
     # Also SEE: https://fedoraproject.org/wiki/Packaging:DistTag
     if ($IsCentOS) {
-        $rpm_dist = "el7.centos"
+        $rpm_dist = "el7"
```

Comment:
I did a quick search and it appears other scripts need to be updated to reflect the name change